sklearn/feature_extraction/text.py

```diff
@@ -909,6 +911,10 @@ def _make_int_array():
     """Construct an array.array of a type suitable for scipy.sparse indices."""
     return array.array(str("i"))
```

Comment:
`str()` is not required here. `array.array("i")` should work both under Python 2 and Python 3.